tests/test_clients.py

```diff
@@ -299,7 +299,7 @@ async def test_s2_only_fields_filtering():
         assert not s2_details.source_quality, "No source quality data should exist"  # type: ignore[union-attr]
 
 
-@pytest.mark.vcr
+@pytest.mark.vcr(record_mode="new_episodes")
```

Comment:
~Don't think you need this~ - I was wrong

```suggestion
@pytest.mark.vcr
```

Reply:
I think we should let the VCR config in `conftest.py` be the one source of truth. So I vote we do actually commit this change